pkg/controller/daemon/update.go

```diff
@@ -172,6 +186,18 @@ func (dsc *DaemonSetsController) rollingUpdate(ctx context.Context, ds *apps.Dae
 				continue
 			default:
 				logger.V(5).Info("DaemonSet pod on node is out of date, this is a surge candidate", "daemonset", klog.KObj(ds), "pod", klog.KObj(oldPod), "node", klog.KRef("", nodeName))
+
+				shouldDeleteDaemonPod, err := dsc.checkShouldDeleteDaemonPod(nodeName, ds)
+				if err != nil {
+					return fmt.Errorf("Couldn't check daemon pod to delete for nodeName %q: %v", nodeName, err)
```

Comment:
lint: error message starts with a lower case

Reply:
Sorry, I've fixed that.